ci/pylint.yml

```diff
@@ -19,4 +19,10 @@ jobs:
         pip install pylint
     - name: Analysing the code with pylint
       run: |
-        pylint `ls -R|grep .py$|xargs`
```

Comment:
It easy if you could make it larger scale or just make it auto

Reply:
Thanks